sections/appendix/metrics.tex

```diff
@@ -0,0 +1,55 @@
+\subsection{Topic coherence}\label{app:topic_coherence}
+The equations, explanations, and values of the hyperparameters in this section are based on~\citet{Syed2017coherence} and the \textit{gensim} python package\footnote{\url{https://radimrehurek.com/gensim/}}.
+Calculating topic coherence requires the following steps:
+
+\begin{enumerate}
+\item Topic-word segmentation into word set pairs
+\item Word and word pair probability calculation
+\item Word set confirmation measure
+\item Aggregation of confirmation measures
+\end{enumerate}
+
+For segmentation, a set of word pairs $S$ is created, which pairs each word in the top-N most probable words $W$ in a specific topic $t$ with all other words in $W$.
+$S$ is defined by \autoref{eq:set_of_word_pairs}.
+
+\begin{equation}\label{eq:set_of_word_pairs}
+	S = \{(W', W*)|W' = \{w_i\};w_i \in W;W* = W\}
+\end{equation}
+
+Before calculating word probabilities, a sliding window of size $s$ where $s =110$ is used to create a set of subdocuments $D_s$ over the document set $D$.
+These subdocuments are used rather than the normal documents to capture some degree of word proximity.
```

Comment:
```suggestion
\todo{talk about how sliding window works, and how it influences this metric}
These subdocuments are used rather than the normal documents to capture some degree of word proximity.
```